### ISSUE TITLE
change facet language looses information

### ISSUE THREAD

**Issue by PietroLiuzzo** [EAGLE-BPN/Inscription-Search-Engine#33]
when switching from one language to another values change. checked on material, ignoratur disappears

**Comment by cprando** [comment]
Don't understand. "Ignoratur" appears if I select "Latin", it disappears if I select another language", it reappears again if I select again Latin or all languages...

**Comment by cprando** [comment]
Btw, we need to change "Defaul language" in "All languages"

**Comment by PietroLiuzzo** [comment]
please, remember that not all values have all languages and this has always
been the case. it ignoratur is removed just because there isn't in the
vocabularies a german translation for it, this gives the user the
experience of not finding reliable data. as a minimum there should be a
warning somewhere that not all labels are in all languages and filters
should fall back on another language to always display the contents
appropriately

On 29 February 2016 at 11:15, cprando notifications@github.com wrote:

> Btw, we need to change "Defaul language" in "All languages"
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/EAGLE-BPN/Inscription-Search-Engine/issues/33#issuecomment-190160204
> .

## 

Pietro Maria Liuzzo
cel (EN): (+44)7529495584
Skype: pietro.liuzzo (Quingentole)
https://uk.linkedin.com/in/pietroliuzzo

**Comment by cprando** [comment]
The behaviour here is the same as in the Vocabularies section. There too if I choose a language where Ignoratur is not translated I cannot view it.
No problem to add a warning. I agree that we need to tell users what they can expect.
If there need to be fallback options, instead, we need to receive a specification how to do this otherwise we cannot implement them. Added Francesco too as this is something new.

**Comment by PietroLiuzzo** [comment]
Yes, the difference is that in the Vocabularies html pages the user is
browsing the terms, so it makes sense that there is no fallback but only
what is there, whilst in the facet this changes what the user is filtering
out from the results.
I would fall back on latin where the language is not given and
alternatively on german, as these are the two most attested (terms coming
from EDR and EDH)

On 29 February 2016 at 11:39, cprando notifications@github.com wrote:

> The behaviour here is the same as in the Vocabularies section. There too
> if I choose a language where Ignoratur is not translated I cannot view it.
> No problem to add a warning. I agree that we need to tell users what they
> can expect.
> If there need to be fallback options, instead, we need to receive a
> specification hiow to do this for this otherwise we cannot implement them.
> Added Francesco too as this is something new.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/EAGLE-BPN/Inscription-Search-Engine/issues/33#issuecomment-190170814
> .

## 

Pietro Maria Liuzzo
cel (EN): (+44)7529495584
Skype: pietro.liuzzo (Quingentole)
https://uk.linkedin.com/in/pietroliuzzo
